packages/@aws-cdk/aws-backup/lib/selection.ts

```diff
@@ -0,0 +1,135 @@
+import * as iam from '@aws-cdk/aws-iam';
+import { Construct, Lazy, Resource } from '@aws-cdk/core';
+import { CfnBackupSelection } from './backup.generated';
+import { BackupableResourcesCollector } from './backupable-resources-collector';
+import { IBackupPlan } from './plan';
+import { BackupResource, TagOperation } from './resource';
+
+/**
+ * Options for a BackupSelection
+ */
+export interface BackupSelectionOptions {
+  /**
+   * The resources to backup.
+   * Use the helper static methods defined on `BackupResource`.
+   */
+  readonly resources: BackupResource[];
+
+  /**
+   * The name for this selection
+   *
+   * @default - a CDK generated name
+   */
+  readonly backupSelectionName?: string;
+
+  /**
+   * The role that AWS Backup uses to authenticate when restoring the target
+   * resource
+   *
+   * @default - a new role will be created
+   */
+  readonly role?: iam.IRole;
+}
+
+/**
+ * Properties for a BackupSelection
+ */
+export interface BackupSelectionProps extends BackupSelectionOptions {
+  /**
+   * The backup plan for this selection
+   */
+  readonly backupPlan: IBackupPlan;
+}
+
+/**
+ * A backup selection
+ */
+export class BackupSelection extends Resource implements iam.IGrantable {
+  /**
+   * The identifier of the backup plan.
+   *
+   * @attribute
+   */
+  public readonly backupPlanId: string;
+
+  /**
+   * The identifier of the backup selection.
+   *
+   * @attribute
+   */
+  public readonly selectionId: string;
+
+  /**
+   * The principal to grant permissions to
+   */
+  public readonly grantPrincipal: iam.IPrincipal;
+
+  constructor(scope: Construct, id: string, props: BackupSelectionProps) {
+    super(scope, id);
+
+    const role = props.role || new iam.Role(this, 'Role', {
+      assumedBy: new iam.ServicePrincipal('backup.amazonaws.com'),
+      managedPolicies: [
+        iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AWSBackupServiceRolePolicyForBackup'),
+        iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AWSBackupServiceRolePolicyForRestores'),
```

Comment:
Taking my comment from my PR, is this policy something we want to include by default?
Maybe adding a method that explicitly sets the policy into the role like `allowRestores` or something similar?

This is because when you set up backup services usually they only require "read" permissions, while "write" is required for restoring, maybe is something we want to provide explicitly from the user? So they can have more control over the service

Reply:
Agree, will look into this.